src/common/src/array/stream_chunk.rs

```diff
@@ -69,6 +69,15 @@ impl Op {
         };
         Ok(op)
     }
+
+    pub fn normalize_update(self) -> Op {
```

Comment:
Please add documentation for non-trivial public interfaces.

Reply:
This is extremely dangerous and I don't think it should be a function of `StreamChunk`